homeassistant/components/fritz/coordinator.py

```diff
@@ -50,6 +51,10 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+FRITZ_DATA_KEY: HassKey[FritzData] = HassKey(DOMAIN)
```

Comment:
Can we improve typing on `FritzData` at the same time?

`current_devices: ValuesView` => `current_devices: ValuesView[set[str]]` on lines 59 and 67

and

```python
    tracked: dict[str, set[str]] = field(default_factory=dict)
    profile_switches: dict[str, set[str]] = field(default_factory=dict)
    wol_buttons: dict[str, set[str]] = field(default_factory=dict)
```